pkg/gcp/apis/provider_spec.go

```diff
@@ -16,12 +16,17 @@ package api
 
 const (
 	// GCPServiceAccountJSON is a constant for a key name that is part of the GCP cloud credentials.
-	GCPServiceAccountJSON string = "serviceAccountJSON"
+	GCPServiceAccountJSON = "serviceAccountJSON"
+	// V1alpha1 is the API version
+	V1alpha1 = "mcm.gardener.cloud/v1alpha1"
```

Comment:
The term `V1alpha1` as s const sounds little confusing, we generally use it to refer to the API-package.

Reply:
Makes sense. I have changed this to - https://github.com/gardener/machine-controller-manager-provider-gcp/pull/5/files#diff-2d6dd4a52e0979ae4fd9992794d1861dR21. I hope it is less confusing now.